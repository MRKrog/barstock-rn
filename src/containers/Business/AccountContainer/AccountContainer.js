import React, { Component } from "react"
import Swiper from "react-native-swiper"
import AccountInfo from "../AccountInfo/AccountInfo"
import BarInventory from "../BarInventory/BarInventory"
import { View, Text, ImageBackground, Dimensions, StyleSheet } from "react-native"
import PastOrder from "../PastOrder/PastOrder";

export default class AccountContainer extends Component{
  render(){
    return(
      <ImageBackground source={require('../../../../assets/bg.png')} style={styles.backgroundImage}>
        <Swiper index={0} loop={false}>
          <AccountInfo />
          <BarInventory />
          <PastOrder />
        </Swiper>
      </ImageBackground>
    )
  }
}

const screenWidth = Dimensions.get("window").width
const screenHeight = Dimensions.get("window").height

const styles = StyleSheet.create({
  MainDisplay: {
    flex: 1,
    width: screenWidth,
    height: screenHeight,
    justifyContent: 'center',
    alignItems: 'center'
  },

  backgroundImage: {
    flex: 1,
    width: screenWidth,
    height: screenHeight,
    resizeMode: 'contain',
    alignItems: 'center',
    justifyContent: 'center',
  },
})