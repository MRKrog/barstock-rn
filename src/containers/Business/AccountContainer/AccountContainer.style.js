import { Dimensions } from "react-native";

const screenWidth = Dimensions.get("window").width;
const screenHeight = Dimensions.get("window").height;

export default {
  MainDisplay: {
    flex: 1,
    width: screenWidth,
    height: screenHeight,
    justifyContent: 'center',
    alignItems: 'center'
  },
  backgroundImage: {
    flex: 1,
    width: screenWidth,
    height: screenHeight,
    resizeMode: 'contain',
    alignItems: 'center',
    justifyContent: 'center',
  },
}