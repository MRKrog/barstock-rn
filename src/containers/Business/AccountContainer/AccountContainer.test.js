import { shallow } from "enzyme"
import { AccountContainer } from "./AccountContainer"
import React from "react"

describe("AccountContainer", () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <AccountContainer />
    )
  })

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot()
  })
})