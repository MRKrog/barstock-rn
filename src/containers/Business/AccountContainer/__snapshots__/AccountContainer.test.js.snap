// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountContainer should match snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../assets/bg.png",
    }
  }
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "height": 1334,
      "justifyContent": "center",
      "resizeMode": "contain",
      "width": 750,
    }
  }
>
  <_default
    automaticallyAdjustContentInsets={false}
    autoplay={false}
    autoplayDirection={true}
    autoplayTimeout={2.5}
    bounces={false}
    disableNextButton={false}
    horizontal={true}
    index={0}
    loadMinimal={false}
    loadMinimalSize={1}
    loop={false}
    onIndexChanged={[Function]}
    pagingEnabled={true}
    removeClippedSubviews={true}
    scrollsToTop={false}
    showsButtons={false}
    showsHorizontalScrollIndicator={false}
    showsPagination={true}
    showsVerticalScrollIndicator={false}
  >
    <Connect(AccountInfo) />
    <Connect(BarInventory) />
    <Connect(PastOrders) />
  </_default>
</ImageBackground>
`;
